homeassistant/components/nordpool/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "nordpool",
+  "name": "Nord Pool",
+  "codeowners": ["@gjohansson-ST"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/nordpool",
```

Comment:
```suggestion
  "documentation": "https://www.home-assistant.io/integrations/nordpool",
  "integration_type": "service",
```

Reply:
Will set to hub according to https://developers.home-assistant.io/docs/creating_integration_manifest#integration-type